### PR TITLE
A first test for commiting

### PR DESCRIPTION
**Pull request by simon-loewe**
#### Adding a comment

Added one line of description comment. Testing for myself the workings of a pull request.

### CODE REVIEW

File: openfisca_survey_manager/scenarios.py [hunk 1]
```diff
@@ -753,6 +753,8 @@ def init_from_data(self, calibration_kwargs = None, inflation_kwargs = None,
                                     Take a look at :func:`build_input_data`
 
         :param data:                Contains the data, or metadata needed to know where to find it.
+
+        :param use_marginal_tax_rate: True to go into marginal effective tax rate computation mode
```
Comment (benjello) [inline openfisca_survey_manager/scenarios.py]: You should also add the type.

Comment (simon-loewe) [reply openfisca_survey_manager/scenarios.py]: Ok, added type!

File: openfisca_survey_manager/scenarios.py [hunk 2]
```diff
@@ -754,7 +754,7 @@ def init_from_data(self, calibration_kwargs = None, inflation_kwargs = None,
 
         :param data:                Contains the data, or metadata needed to know where to find it.
 
-        :param use_marginal_tax_rate: True to go into marginal effective tax rate computation mode
+        :param use_marginal_tax_rate: Bool, True to go into marginal effective tax rate computation mode
```
Comment (benjello) [inline openfisca_survey_manager/scenarios.py]: You should follow http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

Comment (benjello) [reply openfisca_survey_manager/scenarios.py]: Actually I changed it a bit and commited.

**Comment by benjello** [comment]
@simon-loewe : you have to change the version and the changelog and I can merge this PR.

**Comment by benjello** [comment]
@simon-loewe you should also change the version in `setup.py`.

**Comment by benjello** [comment]
@simon-loewe : you can merge (but I am not sure you have the right to do so).

**Pull request merged**